python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -26,29 +30,44 @@ def betweenness_centrality(G, k=None, normalized=True,
     G : cuGraph.Graph
         cuGraph graph descriptor with connectivity information. The graph can
         be either directed (DiGraph) or undirected (Graph)
-    k : int, optional
-        Default is None.
+
+    k : int or list or None, optional, default=None
         If k is not None, use k node samples to estimate betweenness.  Higher
         values give better approximation
+        If k is a list, use the content of the list for estimation
+
     normalized : bool, optional
         Default is True.
         If true, the betweenness values are normalized by
         2/((n-1)(n-2)) for Graphs (undirected), and
         1 / ((n-1)(n-2)) for DiGraphs (directed graphs)
         where n is the number of nodes in G.
-    weight : cudf.Series
+
+    weight : cudf.Series, optional, default=None
         Specifies the weights to be used for each vertex.
```

Comment:
1) Isn't it for each edge instead of vertices? 
2) I think this should be a table containing  ID - weight pairs because of renumbering the code can't assume implicit matching/ordering

Reply:
1. Corrected in 50b27a1
2. How should the user define such table?
Having this as  a boolean option, True: use the weights from the Graph, False: Ignore the weights from the Graph would ensure that the weights are renumbered but the API but would give less flexibility.
If users can define the weights manually how can they get the proper ID - Value mapping when the graph is renumbered? How should we handle the case of Un-directed graphs, as the  edges would get 'duplicated'?

Reply:
> Having this as a boolean option, True: use the weights from the Graph, False: Ignore the weights from the Graph would ensure that the weights are renumbered but the API but would give less flexibility.

That's reasonable I think although different than Nx.

> If users can define the weights manually how can they get the proper ID - Value mapping when the graph is renumbered? How should we handle the case of Un-directed graphs, as the edges would get 'duplicated'?

They provide their own IDs and we renumber before passing. See Weighted Jaccard for instance

Reply:
Udpated the documentation for later, using an cudf.DataFrame so that the users can provide their own IDs.